tests/unit/test_braille/test_normalizeCellArraySize.py

```diff
@@ -0,0 +1,143 @@
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2024 NV Access Limited
+
+"""Unit tests for resizing a cell array before displaying on device."""
+
+import braille
+import unittest
+
+
+class Test_normalizeCellArraySize(unittest.TestCase):
+	"""
+	Tests for BrailleHandler._normalizeCellArraySize.
+	"""
+
+	def test_shrinkSingleLine(self):
+		oldCells = [1, 2, 3, 4, 5, 6, 7, 8]  # fmt: skip
+		expectedNewCells = [1, 2, 3, 4]  # fmt: skip
+		assert braille.handler is not None
+		newcells = braille.handler._normalizeCellArraySize(oldCells, 8, 1, 4, 1)
```

Comment:
could you add param info here? it makes understanding what the tests are doing much easier

```suggestion
		newcells = braille.handler._normalizeCellArraySize(oldCells, oldCellCount=8, oldNumRows=1, newCellCount=4, newNumRows=1)
```